pkg/image/apis/image/v1/helpers.go

```diff
@@ -113,10 +113,12 @@ func ImageWithMetadata(image *Image) error {
 	case 1:
 		image.DockerImageManifestMediaType = schema1.MediaTypeManifest
 
-		if len(manifest.History) == 0 {
```

Comment:
@legionus here: https://github.com/openshift/origin/pull/15624/files#diff-1a18cb8c0d7684cefcef809b9d01aac3R411

we getting this:

```
 image layers: 1, manifest: 0. media type: application/vnd.docker.distribution.manifest.v2+json <nil>" go.version=go1.8.3 instance.id=2226bb84-d159-43e3-9fd2-d3c43ea0a1a7 openshift.logger=registry
```

It seems like the manifest is empty/missing, but the image has 1 layer and the media type... no idea why.